crates/aranya-client/src/client.rs

```diff
@@ -335,11 +337,20 @@ pub struct Team<'a> {
 
 impl Team<'_> {
     /// Adds a peer for automatic periodic Aranya state syncing.
-    pub async fn add_sync_peer(&mut self, addr: Addr, interval: Duration) -> Result<()> {
+    pub async fn add_sync_peer(&mut self, addr: Addr, config: SyncPeerConfig) -> Result<()> {
+        Ok(self
+            .client
+            .daemon
+            .add_sync_peer(context::current(), addr, self.id, config)
+            .await??)
+    }
+
+    /// Sync with peer immediately.
+    pub async fn sync_now(&mut self, addr: Addr, maybe_cfg: Option<SyncPeerConfig>) -> Result<()> {
```

Comment:
In the docs, please describe the difference between `Some` and `None` for `maybe_cfg`.